consolidation/consolidate.go

```diff
@@ -0,0 +1,56 @@
+package consolidation
+
+import (
+	"gopkg.in/raintank/schema.v1"
+)
+
+func Consolidate(in []schema.Point, aggNum uint32, consolidator Consolidator) []schema.Point {
+	num := int(aggNum)
+	aggFunc := GetAggFunc(consolidator)
+	outLen := len(in) / num
+	var out []schema.Point
+	cleanLen := num * outLen // what the len of input slice would be if it was a perfect fit
+	if len(in) == cleanLen {
```

Comment:
It is not obvious that the purpose of the the 2 code paths is simple to handle aggNum dividing into len(in) evenly or not.
Can you add some more comments.